This patch adds the smf, smfproperty and virtualization probes to the list
 of supported probes.
This patch has been submitted upstream, but is being negotiated am hoping
to have it accepted by 2015-Dec-15.

--- openscap-1.2.6/src/OVAL/oval_probe.c.~1~	2015-11-12 10:51:29.490353461 -0800
+++ openscap-1.2.6/src/OVAL/oval_probe.c	2015-11-12 10:54:01.882198119 -0800
@@ -91,7 +91,10 @@
         OVAL_PROBE_EXTERNAL(OVAL_UNIX_FILEEXTENDEDATTRIBUTE, "fileextendedattribute"),
         OVAL_PROBE_EXTERNAL(OVAL_UNIX_GCONF, "gconf"),
         OVAL_PROBE_EXTERNAL(OVAL_UNIX_ROUTINGTABLE, "routingtable"),
-        OVAL_PROBE_EXTERNAL(OVAL_UNIX_SYMLINK, "symlink")
+        OVAL_PROBE_EXTERNAL(OVAL_UNIX_SYMLINK, "symlink"),
+        OVAL_PROBE_EXTERNAL(OVAL_SOLARIS_SMF, "smf"),
+        OVAL_PROBE_EXTERNAL(OVAL_SOLARIS_SMFPROPERTY,"smfproperty"),
+        OVAL_PROBE_EXTERNAL(OVAL_SOLARIS_VIRTUALIZATIONINFO, "virtualizationinfo")
 };
 
 #define __PROBE_META_COUNT (sizeof OSCAP_GSYM(__probe_meta)/sizeof OSCAP_GSYM(__probe_meta)[0])
